includes/class-wc-calypso-bridge-admin-setup-wizard.php

```diff
@@ -107,6 +107,15 @@ public function setup_wizard() {
 
 		// @codingStandardsIgnoreStart
 		if ( ! empty( $_POST['save_step'] ) && isset( $this->steps[ $this->step ]['handler'] ) ) {
+			$current_user = wp_get_current_user();
+			jetpack_tracks_record_event(
```

Comment:
I suppose if we are being ultra paranoid around checking to see if the JS helper is defined we should check here too.

Reply:
Moved this into its own function to dry things up a bit - https://github.com/Automattic/wc-calypso-bridge/pull/133/commits/9e06bcf64eba90d75012341b77a9ffc69d92bc25 . Let me know what you think.